src/lib/response-triggers/help.ts

```diff
@@ -46,9 +47,39 @@ export default class Help extends Trigger {
 	1. I will respond in the room you messaged me in if I am alive.
```

Comment:
Note that line 35 adds an extra level of indentation, at least if the sample on StagingBot currently in the bot testing grounds is accurate.